sdcm/k8s_configs/provisioner/values.yaml

```diff
@@ -12,7 +12,7 @@ common:
   namespace: default
   #
   # Defines whether to create provisioner namespace
-  #
+  # 
```

Comment:
Need to remove trailing space here and below.

Reply:
This is copy paste from their repo, not sure we should do that

Reply:
fixed